web/sites/default/settings.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * Load services definition file.
+ */
+$settings['container_yamls'][] = __DIR__ . '/services.yml';
+
+/**
+ * Include the Pantheon-specific settings file.
+ *
+ * n.b. The settings.pantheon.php file makes some changes
+ *      that affect all environments that this site
+ *      exists in.  Always include this file, even in
+ *      a local development environment, to ensure that
+ *      the site settings remain consistent.
+ */
+include __DIR__ . "/settings.pantheon.php";
+
+/**
+ * Skipping permissions hardening will make scaffolding
+ * work better, but will also raise a warning when you
+ * install Drupal.
+ *
+ * https://www.drupal.org/project/drupal/issues/3091285
+ */
+// $settings['skip_permissions_hardening'] = TRUE;
+
+
+/**
+ * Config Split Settings & Environment Indicator Settings.
+ */
+
+// Disable all config splits by default.
+// $config['config_split.config_split.local']['status'] = FALSE;
+// $config['config_split.config_split.stage']['status'] = FALSE;
+// $config['config_split.config_split.prod']['status'] = FALSE;
+// $config['config_split.config_split.dev']['status'] = FALSE;
+// Set environment indicator - foreground colour.
+$config['environment_indicator.indicator']['fg_color'] = '#FFFFFF';
+
+// Dev.
+if (
+  (isset($_ENV['AH_SITE_ENVIRONMENT']) && $_ENV['AH_SITE_ENVIRONMENT'] == 'dev') ||
+  (isset($_ENV['PANTHEON_ENVIRONMENT']) && $_ENV['PANTHEON_ENVIRONMENT'] == 'dev') ||
+  (getenv('ENVIRONMENT') == 'DEV')
```

Comment:
We could likely extracts these environment checks to a function to improve readability. Something like....

``` php
if (current_environment() == 'dev')
```

Using a `switch` statement would also be easier to read.

Reply:
@RobLoach definitely could. I took this way of doing it from BLT. They keep it fairly up to date and help to integrate with Pantheon and Acquia. When API changes happen they are easy to copy paste this way ;) as environment detection changes from time to time especially if we were to also include environment indicator (example is on config branch) and config split. However please let me know if you still would like me to refactor the code. I'd be open to do it.